ietf/templates/meeting/session_details_panel.html

```diff
@@ -320,51 +320,53 @@ <h3 class="mt-4">Notes and recordings</h3>
-                                <tr>
-                                    <td>
-                                        <a href="{{ r.get_href }}"><i class="bi bi-file-play"></i> {{ r.title }}</a>
-                                    </td>
-                                </tr>
-                            {% endif %}
-                        {% endfor %}
-                    {% endif %}
-                {% endwith %}
-                {% if session.video_stream_url %}
-                    <tr>
-                        <td>
-                            <a href="{{ session.session_recording_url }}">
-                                <i class="bi bi-file-slides"></i> Session recording
-                            </a>
-                        </td>
-                    </tr>
+            {% with session.recordings as recordings %}
+                {% if recordings %}
+                    {# There's no guaranteed order, so this is a bit messy: #}
+                    {# First, the audio recordings, if any #}
+                    {% for r in recordings %}
+                        {% if r.get_href and 'audio' in r.get_href %}
+                            <tr>
+                                <td>
+                                    <a href="{{ r.get_href }}"><i class="bi bi-file-play"></i> {{ r.title }}</a>
+                                </td>
+                            </tr>
+                        {% endif %}
+                    {% endfor %}
+                    {# Then the youtube recordings #}
+                    {% for r in recordings %}
+                        {% if r.get_href and 'youtu' in r.get_href %}
+                            <tr>
+                                <td>
+                                    <a href="{{ r.get_href }}"><i class="bi bi-file-slides"></i> {{ r.title }}</a>
+                                </td>
+                            </tr>
+                        {% endif %}
+                    {% endfor %}
+                    {# Finally, any other recordings #}
+                    {% for r in recordings %}
+                        {% if r.get_href and not 'audio' in r.get_href and not 'youtu' in r.get_href %}
+                            <tr>
+                                <td>
+                                    <a href="{{ r.get_href }}"><i class="bi bi-file-play"></i> {{ r.title }}</a>
+                                </td>
+                            </tr>
+                        {% endif %}
+                    {% endfor %}
                 {% endif %}
+            {% endwith %}
+            {% if session.session_recording_url %}
+                <tr>
+                    <td>
+                        <a href="{{ session.session_recording_url }}">
+                            <i class="bi bi-file-slides"></i>
+                            {% if 'https://meetecho-player.ietf.org/' in session.session_recording_url %}
```

Comment:
Good futureproofing, but the literal should come out of settings. Currently the configuration looks like input for interpolation:
```
MEETECHO_SESSION_RECORDING_URL = "https://meetecho-player.ietf.org/playout/?session={session_label}"
```
I suggest either changing settings.py to 
```
MEETECHO_PLAYER_URL = "https://meetecho-player.ietf.org"
MEETECHO_SESSION_RECORDING_URL = MEETECHO_PLAYER_URL+"/playout/?session={session_label}"
```
and then chainging this if to be 
```
{% if settings.MEETECHO_PLAYER_URL in session.session_recording_url %}
```
or backing the complexity out altogether as YNGNI (when we have something else coming out of the session_recording_url function, this code will likely need to be touched again anyhow).

Reply:
after typing all that, I lean pretty strongly towards saying remove the if and always say Meetecho until we have a need to say otherwise.

Reply:
Concur re: leaving it out.